packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/project/services/ProjectUpdateService.java

```diff
@@ -56,14 +56,13 @@ public IResult<Void> renameProject(UUID projectId, String newName) {
             result = new Failure<>(this.messageService.notFound());
         } else {
             var project = optionalProject.get();
-            project.updateName(newName);
-
-            this.projectRepository.save(project);
+            if (!Objects.equals(project.getName(), newName)) {
+                project.updateName(newName);
+                this.projectRepository.save(project);
+            }
```

Comment:
Testing for this should be done in `Project#updateName`

Reply:
Done.